tests/suites/test_suite_mpi.function

```diff
@@ -926,14 +927,15 @@ void mpi_mod_raw_cond_assign( data_t * input_X,
     size_t bytes = limbs * sizeof( mbedtls_mpi_uint );
     size_t copy_bytes = copy_limbs * sizeof( mbedtls_mpi_uint );
 
+    mbedtls_mpi_mod_modulus_init( &m );
```

Comment:
I suggest getting into the habit of calling init functions immediately after defining the object. We didn't do this historically because it wasn't possible in C89, but we have C99 now.